### DISCUSSION TITLE
How to enable cache server for a distributed team

### DISCUSSION THREAD

**Discussion by XJPan-Jeff** [microsoft/git#478] [Q&A] [answered]
Hello,

We have a team located in US,  India and China. We have a large Azure repo with plenty of binary files (>70GB).  We have LFS enabled.  Our client is Git-2.33.0.vfs.0.0-64-bit.

Our procedure is Scalar clone then sparse check. 

Now the problem is that it is slow to sparse checkout from China side.  I noticed the cache server in git vfs version,  can anyone instruct me that the cache server can help with the performance issue or not,  and how-to?

Many thanks in advance!

Thanks,
Jeff

Thanks,
Jeff

**Comment by derrickstolee** [comment]
Hi @XJPan-Jeff. You will want to contact Azure DevOps support for assistance setting up the cache servers with your instance. Once you have that configured according to their instructions, we can assist if you have issues with your `scalar clone` commands.

**Reply by Wang-XiaoGao** [reply]
@derrickstolee , We have similar request, but I could not find information how to set up a GVFS cache server, for Scalar.
Do you have any clue or document link for us? Thanks a lot.

**Comment by abdurraheemali** [comment, marked as answer]
https://azure.microsoft.com/en-us/support/devops/ for the link to azure devops support
